src/main/java/nl/hsac/fitnesse/util/AllArgScenarioUtils.java

```diff
@@ -0,0 +1,32 @@
+package nl.hsac.fitnesse.util;
+
+import fitnesse.slim.StopTestSlimException;
+import fitnesse.testsystems.TestExecutionException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AllArgScenarioUtils {
```

Comment:
I general I don't think a util method will be very helpful to most fixture developers as they (normally) don't have any plugins installed in FitNesse available on their classpath. But as a sample implementation (maybe include it as sample in the .wiki page) it can be quite helpful.